dist/pom.xml

```diff
@@ -133,6 +134,14 @@
                 </included_buildvers>
             </properties>
         </profile>
+        <profile>
+            <id>databricks</id>
+            <properties>
+                <included_buildvers>
+                    ${databricks.buildvers}
+                </included_buildvers>
+            </properties>
+        </profile>
```

Comment:
how are we building `dist` on CI?  Does it mean we now publish a databricks-only artifact?  If it's just to get the list of databricks versions, is it worth creating a profile if we can just use `-Dexpression=databricks.buildvers` to do this

Reply:
@gerashegalov  `databricks` profile is to get databricks shims aggregating into the dist jar for the Nightly CI, it does NOT mean we publish databricks-only artifact. 

We define this `databricks` profile to better reuse our CI script to get DB shims, https://github.com/NVIDIA/spark-rapids/blob/branch-23.02/jenkins/version-def.sh#L54. 

 Yes we can just use `-Dexpression=databricks.buildvers` to do this, and if you think introducing a new profile is confusing, we can follow it up.

Reply:
ok let us leave it like this for consistency.  we can refactor all these profiles independently of the Databricks 11.3 work